zuul-core/src/test/java/com/netflix/zuul/message/http/CookiesTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+package com.netflix.zuul.message.http;
+
+import com.netflix.zuul.message.Headers;
+import io.netty.handler.codec.http.Cookie;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.List;
+
+import static com.netflix.zuul.message.http.Cookies.cleanCookieHeader;
+import static org.junit.Assert.*;
```

Comment:
please use flat imports.    We have a codestyle xml file you can use in the internal repo.

Reply:
Addressed.